### DISCUSSION TITLE
给PCL2添加简易模式GUI?

### DISCUSSION THREAD

**Discussion by ChiLinCN** [Meloong-Git/PCL#2329] [提问] [answered]
简易模式 顾名思义 在整个PCL界面上只剩下启动器的核心功能区(切换按钮右上角)
只剩下左侧功能区
右侧自定义主页关闭

**Comment by WForst-Breeze** [comment, marked as answer]
你可以通过在 设置-个性化 中关闭自定义主页、开启功能隐藏来实现此功能

Edit: 后续讨论请移步 #2330
